app/javascript/dashboard/routes/dashboard/notifications/components/NotificationsView.vue

```diff
@@ -11,6 +11,7 @@
       <table-footer
         :current-page="Number(meta.currentPage)"
         :total-count="meta.count"
+        :page-size="15"
```

Comment:
The page size is https://github.com/chatwoot/chatwoot/blob/develop/app/finders/notification_finder.rb#L4